src/Nethermind/Nethermind.GitBook/RpcAndCliGenerator.cs

```diff
@@ -189,14 +189,14 @@ private void GenerateDocFileContent(string moduleName, Dictionary<string, Method
                     CreateCliContent(cliBuilder, paramBuilder, returnBuilder, objectsBuilder, cliMethodName, exampleArguments, cliInvocation, methodData);
                 }
             }
-
-            const int notEmptyModule = 20;
-
-            if (rpcBuilder.Length > notEmptyModule)
+            
+            int emptyModuleLength = "this is empty module".Length;
```

Comment:
this was just an example -> I am not sure how the empty module looks
(you had 20 there -> we need an explanation of what 20 is)

Reply:
loool, somehow I wrote something that is exactly 20 characters long but it is just a coincidence...

Reply:
I know, but "this is empty module".Length is exactly 20 and fits perfectly. Empty module has only "# {module_name}" and nothing more